### DISCUSSION TITLE
- No profiles for were found: Xcode couldn't find any iOS App Development provisioning profiles matching

### DISCUSSION THREAD

**Discussion by MonsterDKing** [codemagic-ci-cd/codemagic-docs#2139] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1.- Generate new build

### Expected results

New build completed

### Actual results

I have been working for months without any problem. Just today I got this error, any suggestions, I already checked the xcode and if it has the signed version of iOS since it acquires it through the Apple Connect API and does it in adhoc to distribute for firebase distribution

Archiving xxxxxx...
Updating project for Xcode compatibility.
Upgrading project.pbxproj
Upgrading Runner.xcscheme
Automatically signing iOS for device deployment using specified development team in Xcode project: xxxxx
Running pod install... 24.4s
Running Xcode build...
Xcode archive done. 3.5s
Failed to build iOS app
Error (Xcode): No profiles for 'xxxxx' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'xxxxxxx'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild.
/Users/builder/clone/ios/Runner.xcodeproj

It appears that there was a problem signing your application prior to installation on the device.

Verify that the Bundle Identifier in your project is your signing id in Xcode
open ios/Runner.xcworkspace

Also try selecting 'Product > Build' to fix the problem.
Encountered error while archiving for device.

### Build id (optional)

6536d2d070b1e37fbe3a54df

**Comment by dtrdic** [comment]
Hey @MonsterDKing, I noticed that in your latest build, you are using Xcode version 14.0.1 with Cocoapods 1.13.0, which may not be compatible. Could you try switching to Xcode 15, as this was the version that was running on your previously successful build?

**Reply by MonsterDKing** [reply]
yea in this id change the xcode version but in the lastest give me the same error this is the id to the new build and crash : 

6538114a9bff9046da6e55cf

**Reply by dtrdic** [reply]
can you doublecheck on [Apple developer](https://developer.apple.com/account/resources/profiles/list) side if your provisioning profile has expired, or it's invalid? 
one other observation;
on your successful build the bundleID had this name--> `mx.com.poball.poballapp`
the failed one has this name --> `mx.com.poball.poball`
double-check what bundle id have you set in your xcode project and if it's matching the one you provided in your provisioning profile

**Reply by dtrdic** [reply, marked as answer]
Although i can see some build errors during the Gradle setup;
```
FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* Where:
Script '/Users/builder/programs/flutter/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy' line: 1090

* What went wrong:
A problem occurred evaluating root project 'android'.
> A problem occurred configuring project ':app'.
   > Could not create task ':app:copyFlutterAssetsDebug'.
      > Could not create task ':app:mergeDebugAssets'.
         > Cannot use @TaskAction annotation on method IncrementalTask.taskAction$gradle_core() because interface org.gradle.api.tasks.incremental.IncrementalTaskInputs is not a valid parameter to an action method.
```
If you recently updated your Flutter project try checking gradle and gradle plugin versions to see if the issue is related to the updates, since it sounds as a compatibility issue
